src/components/repo-detail.tsx

```diff
@@ -277,14 +278,20 @@ export function RepoDetail(props: RepoDetailProps) {
 
       <React.Fragment>
         {selectedSha && Boolean(filename) && filesStatus !== "error" && (
-          <JSONDetail
-            key={selectedSha}
+          <DBExplorer
+            sha={selectedSha}
             filename={filename || ""}
             owner={owner as string}
             name={name as string}
-            previousSha={selectedShaPrevious}
-            sha={selectedSha}
           />
+          // <JSONDetail
+          //   key={selectedSha}
+          //   filename={filename || ""}
+          //   owner={owner as string}
+          //   name={name as string}
+          //   previousSha={selectedShaPrevious}
+          //   sha={selectedSha}
+          // />
```

Comment:
Commented out for the time being! I don't have a good sense of how to toggle between these different interfaces.